chef/cookbooks/hyperv/recipes/register_services_ceilometer.rb

```diff
@@ -14,5 +14,5 @@
 service "ceilometer-compute" do
   service_name node[:service][:ceilometer][:name]
   action [:enable, :start]
-  subscribes :restart, "template[#{node[:openstack][:config].gsub(/\\/, "/")}/ceilometer.conf]"
+  subscribes :restart, "template[#{node[:openstack][:ceilometer][:config].gsub(/\\/, "/")}/ceilometer.conf]"
```

Comment:
Metrics/LineLength: Line is too long. [108/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Performance/StringReplacement: Use `tr` instead of `gsub`. (https://github.com/JuanitoFatas/fast-ruby#stringgsub-vs-stringtr-code)